charger/easee.go

```diff
@@ -280,6 +281,50 @@ func (c *Easee) ProductUpdate(i json.RawMessage) {
 		// received observation is outdated, ignoring
 		return
 	}
+
+	switch res.ID {
+		case easee.CHARGER_OP_MODE:
+			//for every op mode change, if last energy updated was triggered more than 3 minutes ago, request new update
+	  		if c.lastEnergyPollTriggered.Before(time.Now().Add(-3 * time.Minute)) {
+				uri := fmt.Sprintf("%s/chargers/%s/commands/%s", easee.API, c.charger, easee.PollLifetimeEnergy)
+				c.Post(uri, request.JSONContent, request.MarshalJSON(nil))  //do not wait for success, fire&forget
```

Comment:
At least error handling should be done, e.g.

```
	if _, err := c.Post(uri, request.JSONContent, request.MarshalJSON(nil)); err != nil {
		//log error
	}
```

Reply:
ok, will add this..